tox.ini

```diff
@@ -20,9 +21,11 @@ deps =
 commands =
     pytest
     make test
+    bash -c 'find . -type f -name "\.coverage\.cocotb" | xargs coverage combine --append'
```

Comment:
I don't think this is even necessary for codecov in CI, but maybe it's helpful locally.